isis/src/base/objs/Stretch/Stretch.cpp

```diff
@@ -38,7 +38,7 @@ namespace Isis {
    * Constructs a Stretch object with default mapping of special pixel values to
    * themselves.
    */
-  Stretch::Stretch() {
+  Stretch::Stretch() : Blob("NAME", "Stretch") {
```

Comment:
In other BLOB subclasses they still have a descriptive name in the default constructor. I would call this ImageStretch or something more descriptive.